examples/daemon-step.yaml

```diff
@@ -53,6 +53,10 @@ spec:
           port: 8086
         initialDelaySeconds: 5
         timeoutSeconds: 1
+    outputs:
+      artifacts:
+        - name: data
+          path: /var/lib/influxdb/data
```

Comment:
Is there another example we can modify or create? This example was nice because it didn't need an artifact repo.